seqr/management/commands/resend_welcome_email_tests.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from django.core.management.base import CommandError
+
+from django.contrib.auth.models import User
+
+TEST_USER_EMAIL = 'test_user@test.com'
+REFERRER_EMAIL = 'test_user_no_staff@test.com'
+
+
+class ResendWelcomeEmailTest(TestCase):
+    fixtures = ['users']
+
+    @mock.patch('seqr.utils.communication_utils.send_welcome_email')
+    def test_command(self, mock_send_welcome_email):
+        call_command('resend_welcome_email', '--email-address={}'.format(TEST_USER_EMAIL),
+                     '--referrer={}'.format(REFERRER_EMAIL))
+        user = User.objects.get(email__iexact=TEST_USER_EMAIL)
```

Comment:
`iexact` is not necessary and makes the code a bit less readable

Reply:
I removed them.